test/e2e/common/node/image_volume.go

```diff
@@ -0,0 +1,122 @@
+limitations under the License.
+*/
+
+package node
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/opencontainers/selinux/go-selinux"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+var _ = SIGDescribe(framework.WithConformance(), "ImageVolume", func() {
+	f := framework.NewDefaultFramework("image-volume")
+	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
+
+	volumeImage := imageutils.GetE2EImage(imageutils.Kitten)
+
+	const (
+		podName             = "test-pod"
+		containerName       = "test-container"
+		volumeName          = "volume"
+		volumePathPrefix    = "/volume"
+		defaultSELinuxUser  = "system_u"
+		defaultSELinuxRole  = "system_r"
+		defaultSELinuxType  = "svirt_lxc_net_t" // that's the SELinux type used by Debian/Ubuntu. Keep it for maximum test compatibility.
+		defaultSELinuxLevel = "s0:c1,c5"
+	)
+
+	createPod := func(ctx context.Context, podName, nodeName string, volumes []v1.Volume, volumeMounts []v1.VolumeMount, selinuxOptions *v1.SELinuxOptions) {
+		pod := &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      podName,
+				Namespace: f.Namespace.Name,
+			},
+			Spec: v1.PodSpec{
+				NodeName:      nodeName,
+				RestartPolicy: v1.RestartPolicyAlways,
+				SecurityContext: &v1.PodSecurityContext{
+					SELinuxOptions: selinuxOptions,
+				},
+				Containers: []v1.Container{
+					{
+						Name:         containerName,
+						Image:        imageutils.GetE2EImage(imageutils.BusyBox),
+						Command:      []string{"/bin/sh", "-c", "while true; do echo test; sleep 1; done"},
+						VolumeMounts: volumeMounts,
+					},
+				},
+				Volumes: volumes,
+			},
+		}
+
+		ginkgo.By(fmt.Sprintf("Creating a pod (%s/%s)", f.Namespace.Name, podName))
+		e2epod.NewPodClient(f).Create(ctx, pod)
+	}
+
+	verifyFileContents := func(podName, volumePath string) {
+		ginkgo.By(fmt.Sprintf("Verifying the volume mount contents for path: %s", volumePath))
+
+		firstFileContents := e2epod.ExecCommandInContainer(f, podName, containerName, "/bin/cat", filepath.Join(volumePath, "data.json"))
+		gomega.Expect(firstFileContents).To(gomega.ContainSubstring("kitten.jpg"))
+
+		secondFileContents := e2epod.ExecCommandInContainer(f, podName, containerName, "/bin/cat", filepath.Join(volumePath, "etc", "os-release"))
+		gomega.Expect(secondFileContents).To(gomega.ContainSubstring("Alpine Linux"))
+	}
+
+	/*
+		Release: v1.35
+		Testname: Image Volume
+		Description: Create a Pod using an image volume and a pull policy of Always.
+		This test verifies that the image volume functionality is available by default and works as intended.
+	*/
+	framework.ConformanceIt("should succeed with pod and pull policy of Always", func(ctx context.Context) {
```

Comment:
I think it against the rules to create a test and promote to conformance right away. For DRA we created a test and promoted in the next release. In this case we may have a chance to promote after code freeze

Reply:
Please revert the file test/conformance/testdata/conformance.yaml

Also remove `ConformanceIt` here, make it a regular test. Another change needed is add the `[MinimumKubeletVersion:1.35]` so version skew tests will not fail next time similar to https://github.com/kubernetes/kubernetes/pull/135134/